.github/workflows/CI.yaml

```diff
@@ -131,8 +131,8 @@ jobs:
             SCHEDULER_COMPATIBLE_K8S_VERSION=${{ env.SCHEDULER_COMPATIBLE_K8S_VERSION }}
           builder: ${{ steps.buildx.outputs.name }}
           push: false
-          cache-from: type=gha, scope=${{ github.workflow }}
-          cache-to: type=gha, scope=${{ github.workflow }}
+          cache-from: type=gha,scope=${{ github.ref_name }}-${{ matrix.config.name }}
```

Comment:
I wonder if `ref_name` is the right selection here. I would assume that we can re-use the cache across branches

Reply:
I changed that because I had security concerns. Think of a malicious branch that gets put into the cache and we re-use that cache on main branch?

Reply:
Fair point! 👍